include/HL_sys_vim.h

```diff
@@ -280,7 +282,7 @@ typedef struct vim_config_reg
                                     |(uint32)((uint32)0U << 6U)\
                                     |(uint32)((uint32)0U << 7U)\
                                     |(uint32)((uint32)0U << 8U)\
-                                    |(uint32)((uint32)0U << 9U)\
+                                    |(uint32)((uint32)1U << 9U)\

```

Comment:
I assume this enables interrupts? Hard to tell...

Reply:
Yes this enables interrupts on that pin. Halcogen shows the configuration nicer